<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Usuarios</class>
 <widget class="QDialog" name="Usuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(226, 226, 226);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_Salvar_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color: rgb(0, 0, 127);
   border-radius: 0px
}

QPushButton:hover{
  background-color:rgb(50,50,50);
  color:rgb(200,200,200)
}

QPushButton:pressed{
  background-color:rgb(35,35,35);
  color:rgb(200,200,200)
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/image/imagens/voltar1.png</normaloff>:/image/imagens/voltar1.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>45</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);</string>
       </property>
       <property name="text">
        <string>Usuarios</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ID:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="btn_Salvar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
   border-radius: 8px
}

QPushButton:hover{
  background-color:rgb(50,50,50);
  color:rgb(200,200,200)
}

QPushButton:pressed{
  background-color:rgb(35,35,35);
  color:rgb(200,200,200)
}
</string>
       </property>
       <property name="text">
        <string>Salvar</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/image/imagens/guardar-dados.png</normaloff>:/image/imagens/guardar-dados.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="L_Id">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="btn_Limpar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
   border-radius: 8px
}

QPushButton:hover{
  background-color:rgb(50,50,50);
  color:rgb(200,200,200)
}

QPushButton:pressed{
  background-color:rgb(35,35,35);
  color:rgb(200,200,200)
}</string>
       </property>
       <property name="text">
        <string>Limpar</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/image/imagens/limpar-limpo (1).png</normaloff>:/image/imagens/limpar-limpo (1).png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="btn_Excluir">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
   border-radius: 8px
}

QPushButton:hover{
  background-color:rgb(50,50,50);
  color:rgb(200,200,200)
}

QPushButton:pressed{
  background-color:rgb(35,35,35);
  color:rgb(200,200,200)
}
</string>
       </property>
       <property name="text">
        <string>Excluir</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/image/imagens/excluir-arquivo.png</normaloff>:/image/imagens/excluir-arquivo.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="L_Data">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="inputMask">
        <string>00/00/0000</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Data</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="btn_Editar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI Black</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
   border-radius: 8px
}

QPushButton:hover{
  background-color:rgb(50,50,50);
  color:rgb(200,200,200)
}

QPushButton:pressed{
  background-color:rgb(35,35,35);
  color:rgb(200,200,200)
}
</string>
       </property>
       <property name="text">
        <string>Editar</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/image/imagens/editar (1).png</normaloff>:/image/imagens/editar (1).png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>45</width>
         <height>45</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Nome:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="L_Nome">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Telefone:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="L_Telefone">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="inputMask">
        <string>(00)00000-0000</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Endereço:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="L_Endereco">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="maxLength">
        <number>80</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Permição</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QComboBox" name="Cbb_Permicao">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Administrador</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Usuário</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Usuario</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="L_Usuario">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Senha</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="L_Senha">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QTableWidget" name="Tab_Usuarios">
     <column>
      <property name="text">
       <string>ID</string>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Data</string>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nome</string>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Telefone</string>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Endereço</string>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Usuario</string>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Senha</string>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Permição</string>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>L_Id</tabstop>
  <tabstop>L_Data</tabstop>
  <tabstop>L_Nome</tabstop>
  <tabstop>L_Telefone</tabstop>
  <tabstop>L_Endereco</tabstop>
  <tabstop>L_Usuario</tabstop>
  <tabstop>L_Senha</tabstop>
  <tabstop>Cbb_Permicao</tabstop>
  <tabstop>btn_Salvar</tabstop>
  <tabstop>btn_Limpar</tabstop>
  <tabstop>btn_Excluir</tabstop>
  <tabstop>Tab_Usuarios</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
